pages/api/search/index.ts

```diff
@@ -0,0 +1,37 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import algoliasearch from "algoliasearch";
+import { getSearchQueries } from "server/search-api-helpers";
+import { P } from "components/MDX";
+
+const APP_ID = process.env.ALGOLIA_APP_ID;
+const API_KEY = process.env.ALGOLIA_API_KEY;
+
+const client = algoliasearch(APP_ID, API_KEY);
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  let rawResults = [];
+  const docsVer = req.headers.referer.includes("ver/")
+    ? req.headers.referer.split("ver/")[1].split("/")[0]
+    : "current";
+
+  try {
+    const { results } = await client.multipleQueries(
```

Comment:
Q: Do we need multiple queries here? I think we only use one now.

Reply:
I created a new issue in Asana. https://app.asana.com/0/1201922182803749/1202064186844923